src/views/accounts/AccountGather/TaskDetail/TaskExecutionList.vue

```diff
@@ -57,6 +57,7 @@ export default {
             width: null
           },
           actions: {
+            width: (this.$i18n.locale === 'pt-br' || this.$i18n.locale === 'en') ? '160px' : '100px',
             formatterArgs: {
               hasDelete: false,
               hasUpdate: false,
```

Comment:
It would be helpful if you could provide the actual code snippet that needs to be checked. The information regarding current date is not relevant.